test/jdk/valhalla/valuetypes/QTypeDescriptorTest.java

```diff
@@ -26,9 +26,9 @@
  * @summary Test core reflection, dynamic proxy and lambdas that generates
  *          classes dynamically that reference Q-type and L-type
  * @compile -XDenablePrimitiveClasses QTypeDescriptorTest.java
- * @run testng/othervm QTypeDescriptorTest
+ * @run testng/othervm -XX:+EnableValhalla -XX:+EnablePrimitiveClasses QTypeDescriptorTest
  * @compile -XDenablePrimitiveClasses QTypeDescriptorTest.java
```

Comment:
This `@compile` seems duplicated.   Since you are on this file, do u mind taking it out?

Reply:
NP